metrics/lib/json.bash

```diff
@@ -82,7 +82,7 @@ EOF
 	# Now add a runtime specific environment section if we can
 	local iskata=$(is_a_kata_runtime "$RUNTIME")
 	if [ "$iskata" == "1" ]; then
-		local rpath="$(command -v $RUNTIME)"
+		local rpath="$(command -v kata-runtime)"
```

Comment:
Out of curiosity, what are we currently passing as "RUNTIME"?

Reply:
@fidencio metrics scripts as they are only using containerd we have as a runtime io.containerd.run.kata.v2

Reply:
Just to completely clear, then "$RUNTIME" in this case is "containerd-shim-kata-v2", and we need to use `kata-runtime` instead.  Do I understand this correctly, @GabyCT?

Reply:
@fidencio exactly we in order to get the kata-env (which give us hypervisor information, image information, etc) we need to use kata-runtime as we can't extract it from containerd-shim-kata-v2